### DISCUSSION TITLE
Welcome to pet-backend Discussions!

### DISCUSSION THREAD

**Discussion by lucasponce** [lucasponce/pet-backend#22] [Q&A] [answered]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by lucasponce** [comment, marked as answer]
This is a comment

**Comment by xeviknal** [comment]
yep, I agree with giving it a try and see where it goes.
